### PR TITLE
Alpha deployment pre-reqs

### PR DESCRIPTION
**Pull request by nadinequinr3**
@davidcurrie Please review. I think it's just the following still outstanding:
* Helm
* Resource Requirements (https://r3-cev.atlassian.net/browse/DOC-4188)

### CODE REVIEW

**Comment by davidcurrie** [comment]
Thanks, @nadinequinr3. For Helm, we should state `3.9.4 or newer`. For the resource requirements, I don't think we're going to have anything meaningful to say about Kafka/Postgres for the Alpha. For Kubernetes, the resource requirements are dependent on the options they select at deployment time so I think we should just drop this section from here.

**Comment by nadinequinr3** [comment]
> Thanks, @nadinequinr3. For Helm, we should state `3.9.4 or newer`. For the resource requirements, I don't think we're going to have anything meaningful to say about Kafka/Postgres for the Alpha. For Kubernetes, the resource requirements are dependent on the options they select at deployment time so I think we should just drop this section from here.

Thanks. Just thinking, should we also list a Docker version?

File: content/en/platform/corda/5.0-alpha/deploying/prerequisites.md [hunk 1]
```diff
@@ -8,24 +8,40 @@ menu:
...
 
-| Software      | Version |
-| ----------- | ----------- |
-| Kubernetes (incl. kubectl)      |     |
-| Helm   |  |
+| Software                                | Version |
+|-----------------------------------------|---------|
+| Kubernetes                              | 1.23    |
+| Amazon Elastic Kubernetes Service (EKS) | 1.23    |
+| Helm | 3.9.4 or newer    |
```
Comment (yarivus) [inline content/en/platform/corda/5.0-alpha/deploying/prerequisites.md:20]: @nadinequinr3 shouldn't we align the columns ( | ) with the above rows ?

**Review by yarivus — COMMENTED** [review]
see my question/comment @nadinequinr3

File: content/en/platform/corda/5.0-alpha/deploying/prerequisites.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-alpha/deploying/prerequisites.md:20]: > @nadinequinr3 shouldn't we align the columns ( | ) with the above rows ?

That last one was added through GitHub so the formatting was lost but Hugo will render them all the same. I'll update anyway to improve readability.

**Comment by nadinequinr3** [comment]
> > Just thinking, should we also list a Docker version?
> 
> On the server side, at least on the cloud platforms, that's typically not something they have any control over. They just get whatever comes with the Kubernetes version they're using (may even be containerd rather than Docker).
> 
> On the client side, they just need Docker to get the images populated into the registry. I can't imagine they'd find a version of Docker that didn't work for that.

OK, makes sense. We need to list it as a pre-requisite to install though, right?

**Review by yarivus — DISMISSED** [review]
LGTM

**Comment by davidcurrie** [comment]
> OK, makes sense. We need to list it as a pre-requisite to deployment though, right?

I'm wondering if it's worth making a split between the target environment (where Kubernetes/Postgres/Kafka are required plus the container registry) and the machine from which the deployment will take place (which requires the Helm client, Kubernetes client, and Docker).

File: content/en/platform/corda/5.0-alpha/deploying/prerequisites.md [hunk 2]
```diff
@@ -8,24 +8,51 @@ menu:
...
 
-| Software      | Version |
-| ----------- | ----------- |
-| Kubernetes (incl. kubectl)      |     |
-| Helm   |  |
+| Software | Version        |
+|----------|----------------|
+| kubectl  | 1.23           |
+| Helm     | 3.9.4 or newer |
+| Docker   |                |
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-alpha/deploying/prerequisites.md:22]: Docker version missing?

File: content/en/platform/corda/5.0-alpha/deploying/prerequisites.md [hunk 3]
```diff
@@ -8,24 +8,51 @@ menu:
...
+| Kubernetes                              | 1.23    |
+| Amazon Elastic Kubernetes Service (EKS) | 1.23    |
+
+### Messaging
+
+| Software                                        | Version |
+|-------------------------------------------------|---------|
+| Kafka                                           | 3.2.0   |
+| Amazon Managed Streaming for Apache Kafka (MSK) | 3.2.0   |
+| Confluent Cloud                                 |         |
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-alpha/deploying/prerequisites.md:39]: Confluent Cound version missing?

File: content/en/platform/corda/5.0-alpha/deploying/prerequisites.md (hunk 2 above)
Comment (davidcurrie) [reply content/en/platform/corda/5.0-alpha/deploying/prerequisites.md:22]: We don't really have a requirement on a particular Docker version.

File: content/en/platform/corda/5.0-alpha/deploying/prerequisites.md (hunk 3 above)
Comment (davidcurrie) [reply content/en/platform/corda/5.0-alpha/deploying/prerequisites.md:39]: As far as we can tell, there is no versioning in Confluent Cloud (beyond its compatibility with particular versions of the Kafka API and the user has no control over these).

**Review by yarivus — COMMENTED** [review]
thanks @davidcurrie & @nadinequinr3

**Pull request merged**